src/Components/CreateImageWizard/steps/ImageOutput/TargetEnvironment.tsx

```diff
@@ -353,9 +349,49 @@ const TargetEnvironment = () => {
             data-testid="checkbox-image-installer"
           />
         )}
-        {supportedEnvironments?.includes('wsl') && wslFlag && (
+        {supportedEnvironments?.includes('wsl') && (
           <Checkbox
-            label="WSL - Windows Subsystem for Linux (.tar.gz)"
+            label={
+              <>
+                WSL - Windows Subsystem for Linux (.tar.gz)
+                <Popover
+                  maxWidth="30rem"
+                  position="right"
+                  bodyContent={
+                    <TextContent>
+                      <Text component={TextVariants.h5}>
+                        WSL is not officially supported by Red Hat.
+                      </Text>
+                      <Text>
+                        Unfortunately Windows Subsystem for Linux is currently
+                        not supported by Red Hat.
+                        <br />
+                        <Button
+                          component="a"
+                          target="_blank"
+                          variant="link"
+                          icon={<ExternalLinkAltIcon />}
+                          iconPosition="right"
+                          isInline
+                          href="https://access.redhat.com/solutions/6338661"
+                        >
+                          Learn more
+                        </Button>
+                      </Text>
+                    </TextContent>
+                  }
+                >
+                  <Button
+                    className="pf-v5-u-pl-sm pf-v5-u-pt-0 pf-v5-u-pb-0"
```

Comment:
Is this necessary? These are from the utilities lib?

Reply:
fair enough! Then I guess they are needed - you use them in other places as well, thanks for checking it out!